rickmorty/pubspec.yaml

```
@@ -0,0 +1,35 @@
+name: rickmorty
+description: "A new Flutter project."
+
+publish_to: 'none' 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)